tests/monotouch-test/ModelIO/MDLMaterialProperty.cs

```diff
@@ -39,6 +44,10 @@ public class MDLMaterialPropertyTest {
 		[TestFixtureSetUp]
 		public void Setup ()
 		{
+#if MONOMAC
+			if (!TestRuntime.CheckMacSystemVersion (10, 11))
```

Comment:
Same: replace `CheckSystemVersion` with call to `TestRuntime.CheckXcodeVersion`.